packages/react/src/components/Calendar/CalendarDay/CalendarDay.tsx

```diff
@@ -1,5 +1,9 @@
+import * as React from 'react';
 import { CalendarDayBase } from './CalendarDay.base';
 import { styles } from './CalendarDay.styles';
 import { styled } from '../../../Utilities';
+import { ICalendarDayProps } from './CalendarDay.types';
 
-export const CalendarDay = styled(CalendarDayBase, styles, undefined, { scope: 'CalendarDay' });
+export const CalendarDay: React.FunctionComponent<ICalendarDayProps> = styled(CalendarDayBase, styles, undefined, {
```

Comment:
This is required to prevent inferred imports in the .d.ts, which API Extractor now complains about.

I think I'm going to separate out this and the other date-time stuff into another PR, because it also included some general types cleanup to more accurately reflect actual usage.